### PR TITLE
password reset link references container instead of host

### PR DESCRIPTION
**Pull request by stugots**
To test:

1. Git checkout resetlink
2. Generate HACKING.md.
2. In the "Postfix" section, start with step # 4.
3. Do a "password reset" sequence.  The link in the e-mail message should work.

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -398,9 +398,16 @@ Then, chown the file so that root owns it:
 
 ### 4. Goldstone-dev.env
 
-In `~/devel/goldstone-server/docker/config/goldstone-dev.env`, add this line:
+In `~/devel/goldstone-server/docker/config/goldstone-dev.env`, add this:
 
     EMAIL_HOST=172.24.4.1
+    EXTERNAL_HOSTNAME=localhost:8000
+
+And then restart your development environment:
```
Comment (lexjacobs) [inline docs/HACKING.md]: lines 406-411 duplicated

Comment (stugots) [inline docs/HACKING.md]: fixed!

**Comment by lexjacobs** [comment]
HACKING.md
line 156. formatting
perhaps should be:
url: **`http://docker-machine ip default:5601/`**

**Comment by lexjacobs** [comment]
Additional lines have been added to `goldstone-dev.env` and `goldstone-test.env`
specifically:
```
EMAIL_HOST=172.24.4.1
EXTERNAL_HOSTNAME=localhost:8000
```

Not currently able to send reset email.

terminal error after 500 response pasted below:


[resetEmail.txt](https://github.com/Solinea/goldstone-server/files/39555/resetEmail.txt)
screen shot 2015-11-19 at 3 08 17 pm
screen shot 2015-11-19 at 3 05 14 pm
[resetEmail.txt](https://files.zenhub.io/564e569c4c4e850173370ad2)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)

**Comment by lexjacobs** [comment]
Awesome. Link worked as-is to get to 'enter new password page'. After resetting password, client redirects to login successfully.

:+1:  (pending formatting issue on line 156 of HACKING.md referenced above)

**Pull request merged**